kogito-business-rules-quarkus/README.md

```diff
@@ -12,6 +12,39 @@ This example shows
 	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
 
+* DRL File
+<p align="center"><img src="docs/images/drl.png"></p>
```

Comment:
I don't love the screenshot of the DRL file. I'd rather paste the contents here, but it's not necessary. Better remove it, as I am pretty sure it will have to change anyway (it's using rule flow groups, ad we don't want to promote that)

Reply:
Removed @evacchi